### PR TITLE
[2201.3.x] Add examples for float langlib 

### LINKED ISSUE #38904: [Improvement]: Add lang library examples for float langlib apidocs
### Description

Currently lang library API docs does not contains usage examples.
So we need to add examples for float lang library API docs.

### Describe your problem(s)

_No response_

### Describe your solution(s)

_No response_

### Related area

-> Other Area

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Fixes #38904

### CODE REVIEW

File: langlib/lang.float/src/main/ballerina/float.bal [hunk 1]
```diff
@@ -113,12 +178,23 @@ public isolated function abs(float x) returns float = @java:Method {
...
 # point being `fractionDigits`; a value of 0 for `fractionDigits` rounds to an integer.
-# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x``.
+# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x`.
 # When `fractionDigits` is 0, this is
 # the same as Java Math.rint method, .NET Math.Round method and
 # IEEE roundToIntegralTiesToEven operation
 # Note that `<int>x` is the same as `<int>x.round(0)`.
 #
+# ```ballerina
+# float f = 3.55;
```
Comment (pcnfernando) [inline langlib/lang.float/src/main/ballerina/float.bal:188]: Same as https://github.com/ballerina-platform/ballerina-lang/pull/39091#discussion_r1051825622

File: langlib/lang.float/src/main/ballerina/float.bal [hunk 2]
```diff
@@ -334,6 +499,14 @@ public isolated function tanh(float x) returns float = @java:Method {
 # - a FloatingPointTypeSuffix is not allowed
 # This is the inverse of function ``value:toString`` applied to an `float`.
 #
+# ```ballerina
+# float:fromString("0.2453") ⇒ 0.2453
+# 
+# float:fromString("-10") ⇒ -10.0
+# 
+# float:fromString("") ⇒ error
```
Comment (pcnfernando) [inline langlib/lang.float/src/main/ballerina/float.bal:507]: Same as https://github.com/ballerina-platform/ballerina-lang/pull/39091#discussion_r1051831363
eg:
```ballerina
# float:fromString("123f") ⇒ error
```

File: langlib/lang.float/src/main/ballerina/float.bal (hunk 1 above)
Comment (SasinduDilshara) [reply langlib/lang.float/src/main/ballerina/float.bal:189]: Updated.

File: langlib/lang.float/src/main/ballerina/float.bal (hunk 2 above)
Comment (SasinduDilshara) [reply langlib/lang.float/src/main/ballerina/float.bal]: Updated

File: langlib/lang.float/src/main/ballerina/float.bal [hunk 3]
```diff
@@ -44,6 +51,13 @@ public isolated function isFinite(float x) returns boolean = @java:Method {
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value
 #
+# ```ballerina
+# float:Infinity.isInfinite() ⇒ true
```
Comment (MaryamZi) [inline langlib/lang.float/src/main/ballerina/float.bal:55]: See https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1052162713

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**